certbot/certbot/compat/os.py

```diff
@@ -60,7 +60,7 @@
 # Basically, it states that appropriate permissions will be set for the owner, nothing for the
 # group, appropriate permissions for the "Everyone" group, and all permissions to the
 # "Administrators" group + "System" user, as they can do everything anyway.
-def chmod(*unused_args, **unused_kwargs):
+def chmod(*unused_args, **unused_kwargs):  # type: ignore
```

Comment:
I personally think it'd be a good idea to include a comment explaining the use of `type: ignore` in this file. I think it probably makes the most sense to just do it once higher up in the file.

Reply:
To be clear, you want to set a global `type: ignore` in the file to not do it for each function declaration, and also add an overall comment about it?

Reply:
I initially was suggesting to keep the `type: ignore` lines local but add a global comment about it since I'm assuming `mypy` is essentially complaining about the same problem so why repeat ourselves, but I think if you want to make it a file level `type: ignore` that's fine too. I think that scope is a little larger than what we really want/need, but only barely and it could save us cycles adding new `type: ignore` lines to satisfy `mypy` in this file in the future.

Reply:
In fact, defining `type: ignore` for the entire module is a way wider scope than what I expected. With it, mypy just does not take account anything from the module, making it raise hundreds of errors of type "certbot.compat.os module does not have method..." everywhere this module is used in the python code.

So I just kepts the local `type: ignore` directives, and added an overall comment about these directives.